pyproject.toml

```diff
@@ -77,6 +77,10 @@ pydantic = {version = ">=1.10,<2.0", optional = true}
 adlfs = {version = ">=2022.4.0", optional = true}
 pyodbc = {version = "^4.0.39", optional = true}
 qdrant-client = {version = "^1.6.4", optional = true, extras = ["fastembed"]}
+ipython = "7.34.0"
```

Comment:
please do not add those to main dependencies. add them as separate group ie. name it ipython and make this group optional. people will install this group only if they need to work on the magics. see other groups that are done like that.

the other thing - to we really need all those dependencies? maybe just ipython is enough? this we'll probably solve if we can write tests for those magics